components/camel-pulsar/src/main/java/org/apache/camel/component/pulsar/PulsarEndpoint.java

```diff
@@ -45,11 +45,17 @@
     @UriPath
     @Metadata(required = true)
     private String topic;
+
+    @UriParam(defaultValue = "true", label = "advanced",
+            description = "Sets whether synchronous processing should be strictly used, or Camel is allowed to use asynchronous processing (if supported).")
```

Comment:
I think the description can be removed, since it's already in the javadoc

Reply:
I have removed the description.